### PR TITLE
Change the email confirmation link

### PR DESCRIPTION
**Pull request by Svitlana-G**

### CODE REVIEW

File: src/actions/accountConfirmation.js [hunk 1]
```diff
@@ -3,10 +3,10 @@ import axios from 'axios'
 const accountConfirmation = async (confirmationCode) => {
   try {
     const result = await axios.get(
-      `http://localhost:3001/api/auth/confirm/${confirmationCode}`,
+      `http://clumsy-glasses-clam.cyclic.app/api/auth/confirm/${confirmationCode}`,
+      // `http://localhost:8080/api/auth/confirm/${confirmationCode}`,
```
Comment (LyubovSugak) [inline src/actions/accountConfirmation.js:7]: do you need commented out code?

File: src/components/user-auth/accountCreated/AccountCreated.tsx [hunk 2]
```diff
@@ -1,29 +1,35 @@
...
 import accountConfirmation from '../../../actions/accountConfirmation'
-import { Box, Button, Typography } from '@mui/material'
+import { Box, Button, Typography, Link, CircularProgress } from '@mui/material'
 import { makeStyles } from 'tss-react/mui'
+import { useParams } from 'react-router-dom'
 
 const AccountCreated = () => {
   const { classes } = useStyles()
-  const [success, setSuccess] = useState(false as boolean | undefined)
+  const [success, setSuccess] = useState(false as boolean)
```
Comment (LyubovSugak) [inline src/components/user-auth/accountCreated/AccountCreated.tsx:10]: Don#t think you need `AS` here

File: src/components/user-auth/accountCreated/AccountCreated.tsx [hunk 3]
```diff
@@ -1,29 +1,35 @@
...
+    fetchData()
+  }, [confirmationCode])
 
   return (
     <>
-      {success ? (
+      {isLoading ? (
+        <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+          <CircularProgress
+            style={{ color: '#FB8F67', position: 'absolute', top: '50%' }}
```
Comment (LyubovSugak) [inline src/components/user-auth/accountCreated/AccountCreated.tsx:29]: `style` plz move to makeStyles hook

File: src/components/user-auth/accountCreated/AccountCreated.tsx [hunk 4]
```diff
@@ -1,29 +1,35 @@
...
+          />
+        </Box>
+      ) : success ? (
```
Comment (LyubovSugak) [inline src/components/user-auth/accountCreated/AccountCreated.tsx:32]: Bad readability to use nested ternary operators, especially in html part. Plz refactor it

File: src/actions/accountConfirmation.js (hunk 1 above)
Comment (Svitlana-G) [reply src/actions/accountConfirmation.js]: I don't need localhost link more. Deleted.

File: src/components/user-auth/accountCreated/AccountCreated.tsx (hunk 2 above)
Comment (Svitlana-G) [reply src/components/user-auth/accountCreated/AccountCreated.tsx]: No, deleted.

File: src/components/user-auth/accountCreated/AccountCreated.tsx (hunk 3 above)
Comment (Svitlana-G) [reply src/components/user-auth/accountCreated/AccountCreated.tsx]: Done

File: src/components/user-auth/accountCreated/AccountCreated.tsx (hunk 4 above)
Comment (Svitlana-G) [reply src/components/user-auth/accountCreated/AccountCreated.tsx]: Done in last commit

**Review by LyubovSugak — APPROVED** [review]

**Pull request merged**